includes/external-connection-cpt.php

```diff
@@ -165,7 +167,10 @@ function setup_list_table() {
 			$sendback = remove_query_arg( array( 'trashed', 'untrashed', 'deleted', 'locked', 'ids' ), wp_get_referer() );
 
 			$deleted  = 0;
-			$post_ids = array_map( 'intval', $_REQUEST['post'] );
+			$post_ids = array();
+			if ( ! empty( $_REQUEST['post'] ) ) {
+				$post_ids = array_map( 'intval', $_REQUEST['post'] );
+			}
```

Comment:
I'm fine with this but we could also make this a one-liner, though maybe it's harder to read?

```
$post_ids = ! empty( $_REQUEST['post'] ) ? array_map( 'intval', $_REQUEST['post'] ) : array();
```